SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -555,38 +555,35 @@ QueryConstraint InterestFactory::CreateAlwaysInterestedConstraint(const AActor*
 	return AlwaysInterestedConstraint;
 }
 
-QueryConstraint InterestFactory::CreateGDKSnapshotEntitiesConstraint() const
+QueryConstraint CreateOrConstraint(const TArray<Worker_ComponentId> ComponentIds)
 {
-	QueryConstraint SnapshotEntitiesConstraint;
-
-	Worker_ComponentId ComponentIds[] = { SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID,
-										  SpatialConstants::VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID,
-										  SpatialConstants::PLAYER_SPAWNER_COMPONENT_ID };
+	QueryConstraint OrConstraint;
```

Comment:
OrContraint? Not sure I understand the name?

Reply:
It's an Or Constraint :)
It's a constraint for a bunch of components.
Would you prefer ComponentOrConstraint ?

Reply:
You mean like binary | or ?

Reply:
A standard or operation - if any of these components match, the query is true.